File: tugas-mingguan-2/main.cpp
// Program C++
#include <iostream>

using namespace std;

int main() {
    // Deklarasi variabel
    int nilai_ujian = 75; // Tipe data integer
    cout << "Nilai UAS : " << nilai_ujian << endl;

    // Penggunaan if else
    if (nilai_ujian > 80) {
        cout << "Skor mu adalah A" << endl;
    } else if (nilai_ujian > 70) {
        cout << "Skor mu adalah B" << endl;
    } else if (nilai_ujian > 60) {
        cout << "Skor mu adalah C" << endl;
    } else {
        cout << "Skor mu adalah D" << endl;
    }

    // Array Satu Dimensi
    int arrSatuDimensi[] = {1, 2, 3, 4, 5};

    // For Loop
    for (int i = 0; i < 5; i++) {
        cout << "Elemen ke-" << i << " = " << arrSatuDimensi[i] << endl;
    }

    // Array Multi Dimensi
    int arrMultiDimensi[2][3] = {{2, 3, 5}, {7, 11, 13}};

    // While Loop
    int j = 0;
    while (j < 3) {
        cout << "Array Multi Dimensi [0][" << j << "] = " << arrMultiDimensi[0][j] << endl;
        j++;
    }

    // Do-while Loop
    int k = 0;
    do {
        cout << "Array Multi Dimensi [1][" << k << "] = " << arrMultiDimensi[1][k] << endl;
        k++;
    } while (k < 3);

    // Input
    int input;
    cout << "Masukkan sebuah angka: ";
    cin >> input;

    // Output
    cout << "Anda memasukkan: " << input << endl;

    // Comment -> Ini single line comment
    /* Ini Multi
       Line Comment */

    return 0;
}

File: tugas-mingguan-2/Main.java
// Program Java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) { //Main Program

        // Deklarasi variabel
        int nilai_ujian = 75; // Tipe data integer
        System.out.println("Nilai UAS : " + nilai_ujian);

        // Penggunaan if else
        if (nilai_ujian > 80) {
            System.out.println("Skor mu adalah A");
        } else if(nilai_ujian > 70){
            System.out.println("Skor mu adalah B");
        }else if(nilai_ujian > 60){
            System.out.println("Skor mu adalah C");
        }else{
            System.out.println("Skor mu adalah D");
        }
        

        // Array Satu Dimensi
        int[] arrSatuDimensi = {1, 2, 3, 4, 5};

        // For Loop
        for (int i = 0; i < 5; i++) {
            System.out.println("Elemen ke-" + i + " = " + arrSatuDimensi[i]);
        }
        

        // Array Multi Dimensi
        int[][] arrMultiDimensi = {{2, 3, 5}, {7, 11, 13}};

        // While Loop
        int j = 0;
        while (j < 3) {
            System.out.println("Array Multi Dimensi [0]["+ j +"] = " + arrMultiDimensi[0][j] );
            j++;
        }
        
        // Do-while Loop
        int k = 0;
        do {
            System.out.println("Array Multi Dimensi [1]["+ k +"] = " + arrMultiDimensi[1][k] );
            k++;
        } while (k < 3);
        

        // Input
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukkan sebuah angka: ");
        int input = scanner.nextInt();

        // Output
        System.out.println("Anda memasukkan: " + input);
        

        // Single line comment
        /* Multi-Line 
           Comment */

        scanner.close();
    }
}